### DISCUSSION TITLE
VP_0.1-pre-alpha.5

### DISCUSSION THREAD

**Discussion by MindustryDagger** [MindustryDagger/VanillaPlus#4] [Announcements]
**(RU)**
- Добавлен тестовый танк
- Сделано оформление
- И да, я не забыл про мод)
- Мод помечается сокращённо как _VanillaPlus_ - **VP**
- **МОД ПОРТИРОВАН НА 137 БИЛД ИГРЫ**

**(EN)**
- added test tank
- Made design
- I didn't forget about mod
- Mod marked abbreviated as _VanillaPlus_ - **VP**
- **MOD HAS BEEN PORTED TO 137 GAME BUILD**

<hr /><em>This discussion was created from the release <a href='https://github.com/MindustryDagger/VanillaPlus/releases/tag/'>VP_0.1-pre-alpha.5</a>.</em>

**Comment by Ardm1** [comment]
Есть некоторые проблемы с заводами такие как завод серебра на энергии , метостекла , и кремния ошибка в том что они ничего не производят.
Когда выйдет следующие обновление ?
Будут ли в моде турели ?
Желаю дальнейшего развития мода !

**Reply by MindustryDagger** [reply]
Обновление уже готовиться, скоро ~через 1-3 недели может позже или раньше. Релиз немного позже.
Турели планируются и будут, также будут силовые проекторы, регенераторы и сверхприводные проекторы. У турелей будут особые механики из V7, несколько типов боеприпасов и звуки из мода (скорее всего). Если что, пишите, отвечу.

**Reply by Ardm1** [reply]
Кстати попробуй сделать в следующем обновлении разгрузщик из меди и свинца со скоростью передачи 4.2

**Reply by MindustryDagger** [reply]
Будет, спасибо за идею

**Reply by Ardm1** [reply]
Можно еще как из одного заброшенного мода сделать ящик 50меди 50свинца и 50 или 80 ед. вместительности

**Comment by Ardm1** [comment]
И у меня такой вопрос у тебя есть дискорд ?

**Reply by Ardm1** [reply]
И кстати у тебя еще завод серебра с такой же ошибкой остался

**Comment by Ardm1** [comment]
![Screenshot_2022-08-27-21-21-18-44](https://user-images.githubusercontent.com/111696754/187043229-e878fd7a-acb4-4ccb-ad78-4f7115236dac.jpg)
Я увидел то что я не должен был видеть если скриншот не получается установить расскажу что я видел

**Reply by MindustryDagger** [reply]
![image](https://user-images.githubusercontent.com/97962472/187060656-98f5247b-f6e0-4dec-b71f-762f94010ef2.png)

**Reply by MindustryDagger** [reply]
[] скобки у тебя не работают. Это баг у тебя

**Comment by MindustryDagger** [comment]
Какая версия игры у тебя?

**Comment by MindustryDagger** [comment]
MindustryDagger#7465 Дискорд

**Reply by Ardm1** [reply]
Запрос отправлен от лица ardm

**Reply by Ardm1** [reply]
Про дискорд я спрашивал что бы тебе предлагать идеи для блоков и ошибки
